ui/components/data-sources/DataSourcesContainer.js

```diff
@@ -83,6 +83,7 @@ class DataSourcesContainer extends Component {
                         selectedDataSource: null })}
                 />
                 <CommonToolbar
+                    hideFilter={false}
```

Comment:
No need, default is false and `onFilter` makes it explicit enough IMO.

Reply:
I mean `hideFilter` will be false by default, no need to add it as a property. Only when it is true like in `ResolversContainer`.